app/views/relation/actions.scala

```diff
@@ -21,12 +21,6 @@ object actions {
       ctx.userId map { myId =>
         (!user.is(myId)) ?? frag(
           !blocked option frag(
-            a(
-              titleOrText(trans.challenge.challengeToPlay.txt()),
-              href := s"${routes.Lobby.home}?user=${user.name}#friend",
-              cls := "btn-rack__btn",
-              dataIcon := ""
-            ),
```

Comment:
why is the challenge button gone from here?

Reply:
I moved it to `app/views/user/show/header.scala` because the buttons here don’t show up when the viewer isn’t logged in.

Reply:
Ah, is this view used elsewhere too? If so, then it is not correct to move it like I did, I see.

Reply:
yes, it's used by the popup on username mousehover